refarch-backend/src/main/java/de/muenchen/refarch/configuration/SpringRestConfiguration.java

```diff
@@ -35,6 +36,7 @@ public void configureRepositoryRestConfiguration(final RepositoryRestConfigurati
      * See {@link RepositoryRestConfigurer#configureValidatingRepositoryEventListener}
      */
     @Override
+    @SuppressFBWarnings(justification = "Values are not further used, so they dont need to be saved in a variable")
```

Comment:
I'm a bit surprised that this works here at all. The `addValidator` method returns a new `ValidatingRepositoryEventListener`, so it doesn't seem to be an in-place operation. So how can this work here at all? Perhaps it would be better to check again whether this is intended by the `RepositoryRestConfigurer`.

Reply:
it looks like, using `addValidator` is supposed to work like that. I haven't found reason or example where the result of it gets used further.

Reply:
Interesting, but strange... :) It's fine.